src/v2/ion/responseTransformer.js

```diff
@@ -77,6 +77,11 @@ const getRemediationValues = (idx) => {
         // messages generically.
         value: [],
       });
+    } else if (idx.context.deviceEnrollment) {
```

Comment:
I am fine this approach, it's reasonable considering end state for device enrollment but still a special type.

Just add a comment here..
// no remediation for device enrollment terminal state